sklearn/utils/estimator_checks.py

```diff
@@ -1296,6 +1301,33 @@ def check_estimators_partial_fit_n_features(name, estimator_orig):
         estimator.partial_fit(X[:, :-1], y)
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_classifier_multioutput(name, estimator_orig):
+
+    estimator_orig = clone(estimator_orig)
+    X, y = make_multilabel_classification(random_state=42)
+    if not (hasattr(estimator_orig, '_label_binarizer') or not
```

Comment:
Why do you need this condition? I would rather name explicitly the estimators that do not handle multioutput string labels (`if name in ["Foo", "Bar]:`). Then we might open an issue to solve these exceptions.

Reply:
Good point! I've changed the condition and pushed. :)
The issue appears to be how `LabelBinarizer` is used by `OneVsRestClassifier` and `RidgeClassifierCV`.
In both cases this error is raised:
```
>           raise ValueError("Multioutput target data is not supported with "
                             "label binarization")
E           ValueError: Multioutput target data is not supported with label binarization
```